File: D4/adv1/src/main.rs
use std::fs;

fn main() {
    let bingo = fs::read_to_string("/home/operaho/Advent/D4/adv.txt").unwrap();
    let f = fs::read_to_string("/home/operaho/Advent/D4/instructions.txt").unwrap();
    let inst: Vec<&str> = f.split(",").collect();
    let mut v0 = Vec::new();
    let mut v1 = Vec::new();
    let (mut c0, mut c1, mut c2): (usize, usize, usize) = (0, 0, 0);

    for i in bingo.lines() {
        if i != "" {
            v0.push(i)
        }
    }

    for j in 0..v0.len() {
        if j == c0 {
            let slice = &v0[c0..c0+5];
            v1.push(slice);
            c0 += 5
        }
    }

    for x in &v1 {
        // snip
    }

    // println!("{:?}", v1;
}

File: D4/adv1/main.py
f0 = open("/home/operaho/Code/Advent/D4/adv.txt", "r")
f1 = open("/home/operaho/Code/Advent/D4/instructions.txt", "r")

bingo, inst, l, fnl, c0 = f0.read().splitlines(), f1.read(), [], [], 0

for i in bingo:
	if i != '':
		l.append(i)


for j in range(0, len(l)):
	if j == c0:
		slc = l[c0:c0+5]
		fnl.append(slc)
		c0 = c0 + 5

for x in fnl:
	print(x)
